keyboards/mechlovin/hannah60rgb/rev2/config.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#define PRODUCT_ID   0x6002
+#define DEVICE_VER   0x0001
+#define PRODUCT      Hannah60 RGB Rev.2
+#define DESCRIPTION  Team.Mechlovin Hannah60 RGB Rev.2
+
+#ifdef RGBLIGHT_ENABLE
+#define RGB_DI_PIN A15
+#define RGBLED_NUM      18
+#define RGBLIGHT_ANIMATIONS
+#define RGBLIGHT_HUE_STEP 8
+#define RGBLIGHT_SAT_STEP 8
+#define RGBLIGHT_VAL_STEP 8
+#endif
+
+//rgb matrix setting// This is a 7-bit address, that gets left-shifted and bit 0
+// set to 0 for write, 1 for read (as per I2C protocol)
+// The address will vary depending on your wiring:
+// 0b1110100 AD <-> GND
+// 0b1110111 AD <-> VCC
+// 0b1110101 AD <-> SCL
+// 0b1110110 AD <-> SDA
+#define DRIVER_ADDR_1 0b1110100
+#define DRIVER_ADDR_2 0b1110110
+#define DRIVER_COUNT 2
+#define DRIVER_1_LED_TOTAL 44
+#define DRIVER_2_LED_TOTAL 34
+#define DRIVER_LED_TOTAL (DRIVER_1_LED_TOTAL + DRIVER_2_LED_TOTAL)
+#define RGB_MATRIX_KEYPRESSES // reacts to keypresses
+#define RGB_MATRIX_KEYRELEASES // reacts to keyreleases (instead of keypresses)
+#define RGB_MATRIX_FRAMEBUFFER_EFFECTS
+#define RGB_DISABLE_AFTER_TIMEOUT 0 // number of ticks to wait until disabling effects
+#define RGB_DISABLE_WHEN_USB_SUSPENDED false // turn off effects when suspended
+#define RGB_MATRIX_LED_FLUSH_LIMIT 16 // limits in milliseconds how frequently an animation will update the LEDs. 16 (16ms) is equivalent to limiting to 60fps (increases keyboard responsiveness)
+#define RGB_MATRIX_MAXIMUM_BRIGHTNESS 255 // limits maximum brightness of LEDs to 200 out of 255. If not defined maximum brightness is set to 255
+#define RGB_MATRIX_STARTUP_MODE RGB_MATRIX_CYCLE_PINWHEEL // Sets the default mode, if none has been set
+
+#define RGB_MATRIX_DISABLE_KEYCODES
```

Comment:
Maybe worth doing: 
```suggestion
#if defined(RGBLIGHT_ENABLE) && defined(RGB_MATRIX_ENABLE)
#    define RGB_MATRIX_DISABLE_KEYCODES
#endif
```

This would better handle some of the logic you want, I think.